### PR TITLE
zoho import password header chnges

### PR DESCRIPTION
**Pull request by balagurusurendar**
Zoho export header changed

attaching new Zoho CSV export file

[zoho_export.zip](https://github.com/bitwarden/jslib/files/4174847/zoho_export.zip)

### CODE REVIEW

File: src/importers/zohoVaultCsvImporter.ts [hunk 1]
```diff
@@ -14,15 +14,15 @@ export class ZohoVaultCsvImporter extends BaseImporter implements Importer {
...
+            if (this.isNullOrWhitespace(value['Password Name'])) {
                 return;
             }
             this.processFolder(result, this.getValueOrDefault(value.ChamberName));
             const cipher = this.initLoginCipher();
             cipher.favorite = this.getValueOrDefault(value.Favorite, '0') === '1';
             cipher.notes = this.getValueOrDefault(value.Notes);
-            cipher.name = this.getValueOrDefault(value['Secret Name'], '--');
-            cipher.login.uris = this.makeUriArray(value['Secret URL']);
+            cipher.name = this.getValueOrDefault(value['Password Name'], '--');
```
Comment (kspearrin) [inline src/importers/zohoVaultCsvImporter.ts]: Let's support both for people who may have an old version of this export.

```
this.getValueOrDefault(value['Password Name'], this.getValueOrDefault(value['Secret Name'], '--'));
```

File: src/importers/zohoVaultCsvImporter.ts [hunk 2]
```diff
@@ -14,15 +14,15 @@ export class ZohoVaultCsvImporter extends BaseImporter implements Importer {
...
+            cipher.login.uris = this.makeUriArray(value['Password URL']);
```
Comment (kspearrin) [inline src/importers/zohoVaultCsvImporter.ts]: Let's support both for people who may have an old version of this export.

```
this.getValueOrDefault(value['Password URL'], this.getValueOrDefault(value['Secret URL']));
```

File: src/importers/zohoVaultCsvImporter.ts [hunk 3]
```diff
@@ -14,15 +14,15 @@ export class ZohoVaultCsvImporter extends BaseImporter implements Importer {
         }
 
         results.forEach((value) => {
-            if (this.isNullOrWhitespace(value['Secret Name'])) {
+            if (this.isNullOrWhitespace(value['Password Name'])) {
```
Comment (kspearrin) [inline src/importers/zohoVaultCsvImporter.ts]: Let's support both for people who may have an old version of this export.

```
            if (this.isNullOrWhitespace(value['Password Name']) && this.isNullOrWhitespace(value['Secret Name'])) {
```

Comment (balagurusurendar) [reply src/importers/zohoVaultCsvImporter.ts]: I think it a good idea

File: src/importers/zohoVaultCsvImporter.ts (hunk 1 above)
Comment (balagurusurendar) [reply src/importers/zohoVaultCsvImporter.ts]: changes added

File: src/importers/zohoVaultCsvImporter.ts (hunk 2 above)
Comment (balagurusurendar) [reply src/importers/zohoVaultCsvImporter.ts]: changes added

File: src/importers/zohoVaultCsvImporter.ts (hunk 3 above)
Comment (balagurusurendar) [reply src/importers/zohoVaultCsvImporter.ts]: changes added

**Pull request merged**